{"name": "test_login_fail[test_info2]", "status": "passed", "attachments": [{"name": "stdout", "source": "176091bc-bf93-4f65-a157-8936e14e013c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_info", "value": "{'mobile': '13502811011', 'pwd': '123123', 'expected': '手机号码或密码错误！请联系管理人员。'}"}], "start": 1611567182475, "stop": 1611567186484, "uuid": "f1521f15-776a-414b-a47e-07e50beacc39", "historyId": "402601c933ab00fb733b14a35e18f3bf", "testCaseId": "912ae1978968c73f58f65a524468c60c", "fullName": "cases.login_test.TestLogin#test_login_fail", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "staging"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20170703AYCC"}, {"name": "thread", "value": "17024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.login_test"}]}